gtsfm/frontend/matcher/twoway_matcher.py

```diff
@@ -34,6 +35,12 @@ def match(
         descriptors_i2: np.ndarray,
         im_shape_i1: Tuple[int, int],  # pylint: disable=unused-argument
         im_shape_i2: Tuple[int, int],  # pylint: disable=unused-argument
+        ratio: Optional[float] = 0.8,
+        cross_check: bool = True,
```

Comment:
Two-way matcher means that cross_check is true. Either 1) rename this class to `SimpleMatcher` or 2) create a new class `OneWayMatcher` which has cross_check false.

Reply:
I could remove the option and always make it cross check? Maybe this class could be changed to `DefaultMatcher` or something more general? It seems unnecessary to have two different classes because of one option (i.e., `cross_check`).